warehouse/query-core/src/main/java/org/apache/commons/jexl2/parser/JexlNodes.java

```diff
@@ -78,6 +79,20 @@ public static <T extends JexlNode> T children(T node, JexlNode... children) {
         return node;
     }
     
+    /**
+     * Sets the supplied child array as the children member of {node} and sets the parent reference of each element in {children} to {node}.
```

Comment:
The documentation here is not quite accurate, as a child array is not actually supplied. Please update this to reflect that a child array will be created from the supplied collection, which is then set as the children of the node.

Reply:
Updated documentation to reflect the new implementation.